test/unit/lib/exceptionalTimesMapper.js

```diff
@@ -21,11 +22,13 @@ describe('exceptionalOpeningTimesMapper', () => {
     });
 
     it('should populate alterations for in date exceptional opening times', () => {
+      tk.travel(new Date('2017', '04', '25'));
```

Comment:
Date constructor parameters should be integers. Might be clearer to extract '4' as a variable called 'May'.